requirements.txt

```diff
@@ -1,6 +1,6 @@
-numpy==1.18
-pyepics==3.4.0
-python-dateutil==2.8.1
-six==1.13.0
-pcaspy==0.7.2
-siriuspy==1.8.0
+numpy>=1.18
+pyepics>=3.4.0
+python-dateutil>=2.8.1
+six>=1.13.0
+pcaspy>=0.7.2
```

Comment:
Isn't it better to keep using `==` in these packages?

Reply:
done.